#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_button_logout.dpatch by Michael Stone <mstone@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: change button logout default to permit easier config via xresources

@DPATCH@
diff -urNad xlockmore-5.27~/xlock/resource.c xlockmore-5.27/xlock/resource.c
--- xlockmore-5.27~/xlock/resource.c	2008-06-08 19:57:28.000000000 -0400
+++ xlockmore-5.31/xlock/resource.c	2009-02-22 13:59:58.448028823 -0500
@@ -282,7 +282,7 @@
 #  if ( USE_BUTTON_LOGOUT <= 0 )
 #   define DEF_BUTTON_LOGOUT "5"	/* User Default, can be overridden */
 #  else
-#   define DEF_BUTTON_LOGOUT "0"	/* User Default, can be overridden */
+#   define DEF_BUTTON_LOGOUT "-1"	/* User Default, can be overridden */
 #  endif
 # endif
 #endif
